lldb/test/API/functionalities/scripted_process/TestScriptedProcess.py

```diff
@@ -60,6 +60,52 @@ def move_blueprint_to_dsym(self, blueprint_name):
         )
         shutil.copy(blueprint_origin_path, blueprint_destination_path)
 
+    def test_missing_methods_scripted_register_context(self):
+        """Test that we only instanciate scripted processes if they implement
```

Comment:
`instanciate` -> `instantiate`

Reply:
Nevermind, apparently this is also an accepted English spelling. No need to change it.